assets/cubyz/blocks/toadstool_stem.zig.zon

```diff
@@ -1,8 +1,9 @@
 .{
```

Comment:
Also please open an issue that these files should be migrated and deleted once item migration is available.